### ISSUE TITLE
Encoders?

### ISSUE THREAD

**Issue by neoOpus** [porcino/Av1ador#7]
Where should I place the encoders and what they are?

**Comment by porcino** [comment]
I don't understand the question, do you mean this? 
![Untitled](https://github.com/porcino/Av1ador/assets/58743577/0e999f29-a6a9-4999-bf76-4995d722d867)
In the bottom bar.

**Comment by neoOpus** [comment]
When I select AV1 nothing happens but encoding to HEVC and AVC works just fine... I am asking if have to place the 3rd party encoders to some location ( I tried but nothing worked so far )

**Comment by porcino** [comment]
All the encoders are included in ffmpeg (bin folder), you don't need anything else. What AV1 encoder did you try?

**Comment by neoOpus** [comment]
I see, but right now, only rav1e is working, others just don't start encoding or crash before… I have decent specs, so maybe something is broken with my computer (my windows is 1 week fresh) or something that needs to be tweaked in the software. 

Furthermore, I will do further investigations and let you know if I find the culprit.

Thank you for developing this

**Comment by porcino** [comment]
What is crashing? the main window (av1ador)? if that's the case, there should be a log file along the executable. If it's the encoder crashing (ffmpeg) av1ador should show you a message with the error, it might be helpful if I could see that.
If you want to run the command on your own, when you start the encode, av1ador creates txt files with the command along the segments (in the temp folder).
![Untitled](https://github.com/porcino/Av1ador/assets/58743577/81775f9d-ca2a-4fa7-8800-6c433b9df625)

**Comment by porcino** [comment]
Closed, since the initial doubt has been clarified, if you still have the crash, open a new issue with the proper log/info.
